src/python/pants/backend/python/dependency_inference/module_mapper.py

```diff
@@ -93,7 +93,13 @@ def find_all_python_projects(all_targets: AllTargets) -> AllPythonTargets:
             first_party.append(tgt)
         if tgt.has_field(PythonRequirementsField):
             third_party.append(tgt)
-    return AllPythonTargets(tuple(first_party), tuple(third_party))
+
+    def _sort_key(tgt: Target) -> str:
+        return tgt.address.spec
```

Comment:
What about making `Target` sortable? It looks like a bunch of the places you found are tuples of targets.

Reply:
I had a fleeting thought about that too.. but the others I think are in address form (as this was the only one needing the key callback)

Reply:
Good call. Done.